cfgov/agreements/management/commands/util.py

```diff
@@ -27,7 +28,7 @@ def upload_to_s3(pdf_obj, s3_key):
 
 
 def get_issuer(name):
-    slug = slugify(name, to_lower=True)
+    slug = slugify(name)
```

Comment:
Unfortunately, there appears to be a slight incompatibility between awesome-slugify's `slugify(to_lower=True)` and Django's `slugify`. I don't want to post the production values here, but if you run this code against the existing `agreements.Issuer` model instances, you'll find three cases where the slugs don't match:

```py
from slugify import slugify
from django.utils.text import slugify as dj_slugify

for i in Issuer.objects.all():
    if slugify(i.name, to_lower=True) != dj_slugify(i.name):
        print(i.name)
```

Obfuscated, these are banks that have special characters, e.g. `W.B. Bank` and `W&B Bank`. Using awesome-slugify's logic, these both become `w-b-bank`. Using Django's logic, they become `wb-bank`.

It appears this difference is because [Django](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.text.slugify)

> Removes characters that aren’t alphanumerics, underscores, or hyphens.

but awesome-slugify doesn't do that.

In practice, what this would mean immediately is that some existing URLs (like https://www.consumerfinance.gov/credit-cards/agreements/issuer/m-t-bank/) would break and be replaced by slightly different ones (https://www.consumerfinance.gov/credit-cards/agreements/issuer/mt-bank/). The agreements would still be discoverable through the search [here](https://www.consumerfinance.gov/credit-cards/agreements/), but if someone happened to bookmark a page, it would break.

Whether this is significant enough to hold this change back is a question I will defer to stakeholders about.

Reply:
Would it make sense to import the awesome-slugify `slugify()` function instead then? This is a conflict we have right now. (I'm not sure why tox-pip-extensions doesn't complain about it... maybe because they're specified in the same requirement file right now?)

Reply:
I've updated my comment above. @higs4281 correctly points out that this won't affect live URLs, but will only affect things the next time the data is loaded quarterly. This change still means that those bank URLs will change slightly upon the next load.